.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: php
 
-php: [5.4, 5.5, 5.6, 7.0, hhvm]
+php: [7.1]
 
 matrix:
     include:
+        # Test with default configuration
+        - { php: 7.1 }
```

Comment:
Without this I had only 2 jobs. See: https://travis-ci.org/swarrot/swarrot/builds/244940684

It looks OK right now: https://travis-ci.org/swarrot/swarrot/builds/244942392

Any idea of what's wrong in the config?